lib/nrf_modem_lib/nrf_modem_os.c

```diff
@@ -509,11 +511,9 @@ void trace_uart_init(void)
 #define RTT_BUF_SZ		(CONFIG_NRF_MODEM_LIB_TRACE_MEDIUM_RTT_BUF_SIZE)
 static int trace_rtt_channel;
 static char rtt_buffer[RTT_BUF_SZ];
-#endif
```

Comment:
That way, `trace_rtt_init()` will no longer be available if `NRF_MODEM_LIB_TRACE_ENABLED=y` and UART is used for the medium. Would need more sophisticated ifdef slicing when calling initializers (it's either UART or RTT as a medium choice, never both).

Reply:
Good catch.

Fixed.